fox_scraper/fox_scraper.py

```diff
@@ -48,63 +72,54 @@ def __init__(self, site_address):
     def get_web_content(self, http_address):
```

Comment:
im not sure im following, this function gets `http_address` which suggest it gets an url not the web content...